cmd/juju/action/status.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/juju/juju/apiserver/params"
 	"github.com/juju/juju/cmd/modelcmd"
 	"github.com/juju/juju/cmd/output"
+	"time"
```

Comment:
Going to add some output tests. You are very correct they are missing and I can use this opportunity to add them. Thank you @jameinel